package.json

```diff
@@ -245,7 +245,10 @@
 		"clean:build": "npx rimraf build server/bundler/*.json .babel-cache",
 		"clean:devdocs": "npx rimraf server/devdocs/search-index.js server/devdocs/proptypes-index.json server/devdocs/components-usage-stats.json",
 		"clean:packages": "npx lerna run clean",
-		"clean:public": "npx rimraf public/*.{css,css.map} public/fallback/*.{css,css.map,js,js.map} public/evergreen/*.{css,css.map,js,js.map} public/fallback/images/*.{gif,jpg,png,svg} public/evergreen/images/*.{gif,jpg,png,svg} public/fallback/{sections,sections-rtl,vendors~*} public/evergreen/{sections,sections-rtl,vendors~*}",
+		"clean:public-evergreen": "npx rimraf 'public/evergreen/*.{css,css.map,js,js.map}' 'public/evergreen/images/*.{gif,jpg,png,svg}' 'public/evergreen/{sections,sections-rtl,vendors~*}'",
```

Comment:
can this be simplified to `npx rimraf 'public/evergreen'`?

Reply:
As it stands, it leaves the evergreen folder in place in public. It would be nice to remove it entirely.

Reply:
> can this be simplified to `npx rimraf 'public/evergreen'`?

I'm not sure. When working on the multiple builds PR, I assumed there was some hidden complexity there that explained why `clean:public` wasn't simply `npx rimraf public` and mimicked that into the new subdirectories.

I'm happy to switch to your proposal if you think it's safe.

Reply:
yeah, `public` itself is ... special. Try removing the whole directory and then do a `git status`. It's a mix of built files and checked in files, which is a bummer.

Reply:
and blowing away `evergreen` and `fallback` entirely should be safe.

Reply:
Got it. Happy to make the change, then.

Perhaps we should consider renaming the repo version of `public` to `static`, and simply copying the files there to `public` during build? That would allow us to make `build` a fully disposable directory.

Reply:
Actually, if you're OK with that, I'll drop this PR and prepare another one with what I mentioned 😁